Extension/src/Debugger/configurationProvider.ts

```diff
@@ -274,7 +274,6 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
             const outputChannel: logger.Logger = logger.getOutputChannelLogger();
             outputChannel.appendLine(localize("debugger.launchConfig", "Launch configuration:"));
             outputChannel.appendLine(JSON.stringify(config, undefined, 2));
-            logger.showOutputChannel();
```

Comment:
So if you remove that, the user won't see the log and We want to show the logging window.  I think this is a hack so you should mark it with a bug number and comment it out and file a bug with VS Code to fix the problem.

Reply:
There a weird fight between the terminal window and output window in the "Build and Debug Active File" scenario. It flashes the output window then goes to terminal for compiling.